demos/.prettierc

```diff
@@ -0,0 +1,9 @@
+{
+    "printWidth": 100,
+    "tabWidth": 4,
```

Comment:
you are writing a lot of code that represents deeply nested objects, I would consider using tabWidth: 2